app/javascript/turbo_streams/index.js

```diff
@@ -0,0 +1,13 @@
+function updateSessionStorageItem() {
```

Comment:
**nitpick:** You won't need this anymore

Reply:
Please see above comment.